src/main/java/seedu/duke/models/schema/Student.java

```diff
@@ -0,0 +1,46 @@
+package seedu.duke.models.schema;
+
+public class Student {
+
+    private String name;
+    private Major major;
+
+    public Student(String name, Major major) {
+        this.name = name;
+        this.major = major;
+    }
+
+    public Student() {
+        this.name = null;
+        this.major = null;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public Major getMajor() {
+        return major;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void setMajor(Major major) {
+        this.major = major;
+    }
+
+    public String updateMajor(String userInput) {
+        String[] words = userInput.split(" ");
+        if (words.length < 2) {
+            return "currentMajor";
+        }
```

Comment:
Consider abstracting this into a clearer method to illustrate what you are trying to achieve.